client/db/bolt/testdata/blueprint.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2020 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
```

Comment:
Wrong license.
Out of curiosity, is this file based on something from wallet or node?

Reply:
That was sourced from dcrpool and updated, dcrwallet is the origin of the upgrade infrastructure however.